### PR TITLE
Epicenter 894 login styles

### PR DESCRIPTION
**Pull request by andrewnatt**

### CODE REVIEW

**Comment by narenranjit** [comment]
@andrewnatt content is fine, but `dist` is the generated files. The source is in `src/components/login`. Can you re-submit against that?

File: src/components/login/index.html [hunk 1]
```diff
@@ -7,15 +7,16 @@
...
         <meta name="viewport" content="width=device-width, initial-scale=1">
 
-        <!-- login.css provides styling for the group selector pop over dialog -->
-        <link rel="stylesheet" href="https://forio.com/tools/js-libs/<%= version %>/components/login/login.css">
-
         <!-- the inserted pop over is compatible with bootstrap classes, so optionally you can include bootstrap -->
         <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.2/css/bootstrap.min.css">
+
+        <!-- login.css provides styling for the group selector pop over dialog -->
+        <link rel="stylesheet" href="https://forio.com/tools/js-libs/1.5.0/components/login/login.css">
```
Comment (jaimedp) [inline src/components/login/index.html]: this should be <%= version %> instead of 1.5.0

File: src/components/login/index.html [hunk 2]
```diff
@@ -7,15 +7,16 @@
...
+        
         <!-- make sure to include jquery, epicenter.js and login.js -->
         <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-        <script src="https://forio.com/tools/js-libs/<%= version %>/epicenter.min.js"></script>
-        <script src="https://forio.com/tools/js-libs/<%= version %>/components/login/login.js"></script>
+        <script src="https://forio.com/tools/js-libs/1.5.0/epicenter.min.js"></script>
```
Comment (jaimedp) [inline src/components/login/index.html]: this should be <%= version %> instead of 1.5.0

File: src/components/login/index.html [hunk 3]
```diff
@@ -7,15 +7,16 @@
...
+        <script src="https://forio.com/tools/js-libs/1.5.0/components/login/login.js"></script>
```
Comment (jaimedp) [inline src/components/login/index.html]: this should be <%= version %> instead of 1.5.0

**Pull request merged**